docs/usage_stats_collection.md

```diff
@@ -0,0 +1,54 @@
+# Usage Stats Collection
+
+Skyplane collects usage stats by default. All data is anonymous and helps us maintain an open-source map of cloud network performance data. This data is used to improve the speed of your transfers by routing your traffic around slow links. It will only be used by the Skyplane team to improve the services and for research purposes.
+
+Here are the guiding principles of our collection policy:
+
+- **No surprises** — you will be notified before we begin collecting data. You will be notified of any changes to the data being collected or how it is used.
+- **Easy opt-out:** You will be able to easily opt-out of data collection.
+- **Transparency** — you will be able to review all data that is sent to us.
+- **Control** — you will have control over your data, and we will honor requests to delete your data.
+- We will **not** collect any personally identifiable data or proprietary code/data.
+- We will **not** sell data or buy data about you.
+
+## What data is collected
```

Comment:
Explicitly list everything we collect here as bullet points. We shouldn't route people to `UsageStatsToReport` but instead also point them to `/tmp/...`